doc/source/explanation/index.rst

```diff
@@ -1,15 +1,15 @@
-What is Girder?
-===============
+What is Resonant?
+=================
 
 .. toctree::
    :hidden:
 
-   girder4
+   resonant
 
 
-**Girder 4** is the Kitware Data and Analytics team’s newest version of the framework we use to create rich web applications and APIs tailored to specific kinds of data and tasks. Girder 4 is the direct successor to Girder 3, which used the CherryPy web server framework to present a monolithic platform upon which to build specific applications. Girder 4 marks a technology shift that enables us to build those applications more effectively: it is based instead upon Django, a more expansive web application framework with out-of-the-box support for many of the nuts and bolts of web application engineering. Django’s integrated support for domain models backed by a comprehensive object-relational mapping (ORM) means that Girder 4 projects place *domain-driven design* front and center, enabling our team to better understand and address the computing and data problems we take on, so we can deliver the right solution every time.
+**Resonant** is the Kitware Data and Analytics team’s newest framework we use to create rich web applications and APIs tailored to specific kinds of data and tasks. Girder, another framework created by the Data and Analytics team, used the CherryPy web server framework to present a monolithic platform upon which to build specific applications. Resonant marks a technology shift that enables us to build those applications more effectively: it is based instead upon Django, a more expansive web application framework with out-of-the-box support for many of the nuts and bolts of web application engineering. Django’s integrated support for domain models backed by a comprehensive object-relational mapping (ORM) means that Resonant projects place *domain-driven design* front and center, enabling our team to better understand and address the computing and data problems we take on, so we can deliver the right solution every time.
```

Comment:
Please look over this paragraph in particular.

Reply:
Looks fine, except for one grammar suggestion.